recipe/meta.yaml

```diff
@@ -4,12 +4,12 @@ package:
   name: ucx-split
 
 source:
-  url: https://github.com/openucx/ucx/archive/v{{ version }}.tar.gz
-  sha256: bf1c1ce6583ea1ab805feceb000157dc1fb825a60ddac0fbd9a329a010f9ad72
+  url: https://github.com/openucx/ucx/releases/download/v{{ version }}/ucx-{{ version }}.tar.gz
+  sha256: f73770d3b583c91aba5fb07557e655ead0786e057018bfe42f0ebe8716e9d28c
 
 build:
   skip: true  # [not linux or cuda_compiler_version == "None"]
-  number: 3
+  number: 4
```

Comment:
JFYI bumped the `build/number` as we are changing the package while keeping the same `version` number

Reply:
Oh right. I actually had it updated later locally, but forgot to push the updated commit.